core-web/libs/template-builder/src/lib/components/template-builder/store/template-builder.store.ts

```diff
@@ -33,15 +33,26 @@ export class DotTemplateBuilderStore extends ComponentStore<DotTemplateBuilderSt
     // We need to discuss how we will save this to not trigger the parse every time
 
     public items$ = this.select((state) => state.items);
+    public containerMap$ = this.select((state) => state.containerMap);
```

Comment:
Maybe we don't need this.

Reply:
Do we need it?

Reply:
The one here is used to pass it to vm$ as are `items$` and `layoutProperties$`, does that make sense @fmontes, @zJaaal?

Reply:
Yeah, we made a mistake there, we don't need to pass the selectors.